ksqldb-parser/pom.xml

```diff
@@ -84,6 +84,18 @@
             <version>${io.confluent.ksql.version}</version>
             <scope>test</scope>
         </dependency>
+      <dependency>
```

Comment:
Do you actually need this? If you know how to do it, we should depend on one version of approval tests throughout (via ksql-test-util or parent POM), if not let me know and I'll task myself up with that

Reply:
You are right, I don't need it. I removed it now :)